deegree-core/deegree-core-coverage/src/main/java/org/deegree/coverage/raster/cache/RasterCache.java

```diff
@@ -481,6 +486,7 @@ public RasterReader addReader( RasterReader reader ) {
                 boolean createCache = reader.shouldCreateCacheFile();
                 File cacheFile = null;
                 if ( createCache ) {
+                    LOG.trace( "create cachefule for location {}", reader.getDataLocationId() );
```

Comment:
minor typo in "cachefule" to "cachefile"

Reply:
fixed in https://github.com/deegree/deegree3/pull/1084/commits/61cc5eb3ca48965babd2eac22196db3d6ebf30db